src/python/grpcio/grpc/__init__.py

```diff
@@ -299,6 +299,23 @@ def add_callback(self, callback):
     raise NotImplementedError()
 
 
+############################  gRPC Metadata  ##################################
+
+
+class Metadatum(six.with_metaclass(abc.ABCMeta, tuple)):
```

Comment:
Because clients provide Metadata and we don't offer a concrete implementation, this allows them to provide a tuple in the place of Metadata and still adhere to the type spec.